### PR TITLE
Validate the extension id 

### PR DESCRIPTION
**Pull request by pajaydev**
**Changes**
 - Validate the extension id (allowing only lowercase characters), once the user presses enter or click on Add button.
 - Shows the error message if the input is invalid.

Fixes https://github.com/manojVivek/responsively-app/issues/240

Screen Shot 2020-07-07 at 10 16 23 PM

### CODE REVIEW

**Comment by jjavierdguezas** [comment]
Are we not going to check that it must be 32 characters long?

**Comment by pajaydev** [comment]
@jjavierdguezas ohh let me include that.

**Comment by pajaydev** [comment]
@jjavierdguezas  Shall we have seperate error messages for both the case (invalid char and length) ??.

**Comment by jjavierdguezas** [comment]
hi @pajaydev , maybe we should discuss this on gitter with @manojVivek , I think that one general error will do fine `Extension Id must have 32 lower case alphabets` or something like that

File: desktop-app/app/components/ExtensionsManager/index.js [hunk 1]
```diff
@@ -105,7 +105,8 @@ export default function ExtensionsManager({triggerNavigationReload}) {
   };
 
   const validateExtensionId = extensionId =>
-    extensionId.match(/^[a-z]+$/) && extensionId.length <= 32;
+    (extensionId.match(/^[a-z]+$/) && extensionId.length <= 32) ||
```
Comment (jjavierdguezas) [inline desktop-app/app/components/ExtensionsManager/index.js:108]: @pajaydev should'n it be `extensionId.length === 32` ?

Comment (pajaydev) [reply desktop-app/app/components/ExtensionsManager/index.js]: Fixed it

**Comment by manojVivek** [comment]
@pajaydev Thanks for fixing this, merging this in now. Looking forward to more like this! 🙌

**Comment by manojVivek** [comment]
@all-contributors Please add @pajaydev for code.

**Comment by manojVivek** [comment]
@pajaydev Can you let us know your Twitter handle? We would love to give a shout out for your work from our twitter account.

**Comment by pajaydev** [comment]
@manojVivek  This is my Twitter handle @pajay2507. Thanks for doing this, I appreciate it 👍

**Pull request merged**